components/src/Input/Input.js

```diff
@@ -42,6 +42,7 @@ const StyledInput = styled.input(
     fontSize: theme.fontSizes.medium,
     fontFamily: theme.fonts.base,
     lineHeight: theme.lineHeights.base,
+    margin: theme.space.none,
```

Comment:
Setting margin here after space is above prevents the use of the "m" space props. One solution would be to move space below these styles